qp_klp/klp.py

```diff
@@ -246,6 +246,10 @@ def sequence_processing_pipeline(qclient, job_id, parameters, out_dir):
                 csv_fps.append(join(root, csv_file))
 
         for project, upload_dir in special_map:
+            if sifs:
+                # move uncompressed sifs to upload_dir.
+                cmds.append(f'cd {out_dir}; mv {tmp} {upload_dir}')
```

Comment:
Note that this will move _all_ files to the first upload_dir, and it should be one per project.

Reply:
🤦